web/src/app/schedules/ScheduleDetails.js

```diff
@@ -46,6 +46,16 @@ export default function ScheduleDetails({ scheduleID }) {
   const [configTempSchedule, setConfigTempSchedule] = useState(null)
   const [deleteTempSchedule, setDeleteTempSchedule] = useState(null)
 
+  const onNewTempSched = useMemo(() => () => setConfigTempSchedule(true), [])
```

Comment:
Use `useCallback` for these
```suggestion
  const onNewTempSched = useMemo(() => setConfigTempSchedule(true), [])
```